applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -74,6 +76,51 @@ The sensor module will only send out a :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY`
    This is beyond the scope of the general asset tracker framework this application provides.
    Therefore, the readings are not transmitted to the cloud and are only used to detect a binary active and inactive state.
 
+Bosch Software Environmental Cluster (BSEC) library
+===================================================
+
+The sensor module supports integration with the Bosch BSEC signal processing library via the *external sensors* internal convenience API.
+If enabled, the BSEC library is used instead of the BME680 Zephyr driver to provide sensor readings from the BME680 for temperature, humidty and atmospheric pressure.
+In addition, the BSEC driver provides an additional sensor reading, air quality (Indoor-Air-Quality AIQ), which is a metric given in the range of 0-500 that estimates the air quality of the
+surrounding climate.
+
+Due to the BSEC library requiring separate lisencing, it is not a default part of NCS, but can be downloaded externally and imported into the NCS source tree.
+To enable Bosch BSEC perform the following steps:
+
+1. Download the Bosch BSEC library, available through the following link `Bosch BSEC`_.
+#. Place the content of the library in the path given by `CONFIG_EXTERNAL_SENSORS_BME680_BSEC_PATH` or update the path configuration to reference the location of the library.
+#. Disable the Zephyr BME680 driver by setting :kconfig:`CONFIG_BME680` to false.
+#. Enable the *external sensors* API BSEC integration layer by enabling `CONFIG_EXTERNAL_SENSORS_BME680_BSEC`.
+
+Air quality readings will be provided with the :c:enum`SENSOR_EVT_ENVIRONMENTAL_DATA_READY` event.
+
+External sensors API BSEC configurations
+----------------------------------------
+
+.. _CONFIG_EXTERNAL_SENSORS_BME680_BSEC
+
+CONFIG_EXTERNAL_SENSORS_BME680_BSEC - Use Bosch BSEC library for the BME680.
+
+.. _CONFIG_EXTERNAL_SENSORS_BME680_BSEC_PATH
+
+CONFIG_EXTERNAL_SENSORS_BME680_BSEC_PATH - Path of the Bosch BSEC library folder.
+
+.. _CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_ULTRA_LOW_POWER
+
+CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_ULTRA_LOW_POWER - BSEC ultra low power sample rate.
+
+.. _CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_LOW_POWER
+
+CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_LOW_POWER - BSEC low power sample rate.
+
+.. _CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_HIGH_POWER
+
+CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_HIGH_POWER - BSEC ultra high power sample rate.
```

Comment:
```suggestion
.. _CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_HZ_CONTINUOUS

CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_RATE_HZ_CONTINUOUS - BSEC sample rate in case of Continuous Mode in Hertz (Hz).
```

Reply:
Third mode is named `continuous`.